app/mailers/user_mailer.rb

```diff
@@ -353,11 +353,11 @@ def prompter_notification(work_id, collection_id=nil)
   def delete_work_notification(user, work)
     @user = user
     @work = work
-    work_copy = generate_attachment_content_from_work(work)
-    work_copy = ::Mail::Encodings::Base64.encode(work_copy)
-    filename = work.title.gsub(/[*:?<>|\/\\\"]/,'')
-    attachments["#{filename}.txt"] = { content: work_copy, encoding: "base64" }
-    attachments["#{filename}.html"] = { content: work_copy, encoding: "base64" }
+    @download = Download.new(@work, mime_type: "text/html")
```

Comment:
You can use a local variable for this. Instance variables are accessible from views, but we don't need that here.